src/index.js

```diff
@@ -180,17 +185,22 @@ function printDataByKey(costObject, key) {
 
 async function runExplorer() {
   const rawCost = await getRawCosts();
+
   const assignedCost = await getAssignedCost(rawCost);
+
   const groupedByRegion = sumByKey(assignedCost, 'region');
   const groupedByMonth = sumByKey(assignedCost, 'month');
+  const groupedByService = sumByKey(assignedCost, 'service');
```

Comment:
I'm pretty happy with this interface. It was a really easy change to make here to add breakdown by another key.